aims/aim3.md

```diff
@@ -0,0 +1,60 @@
+# Aim 3:
+
+**To evaluate the extent to which latent space arithmetic works in HCA benchmark datasets generated under this or other initiatives.**
+
+## Background:
+
+Generative models can approximate a lower dimensional manifold.
+These manifolds constrain datasets to occupy a small subset of total space and can contain vital information regarding data generating functions.
+Remarkably, certain classes of generative models, including variational autoencoders (VAE) and generative adversarial networks (GANs), that add a distribution matching constraint, imbue intuitive mathematical features to the learned latent features [1,2].
+For instance, a recent GAN learned latent representations that could be manipulated with arithmetic: Subtracting out the essence of a smile from a woman and adding it to a neutral man resulted in an image vector of a smiling man [3].
+While image data have been extensively modeled, less is known about the learning capacity of such models applied to single cell gene expression data.
+The _objective of this aim_ is to assess the extent to which single cell gene expression latent spaces can be manipulated and how such manifold preserving properties will enable biological discovery regarding cell type, cellular differentiation, and disease perturbation.
+The _working hypothesis of this aim_ is that the latent space will preserve arithmetic operations between state transitions of differentiating hematopoietic cells.
+
+## Preliminary Data:
+
+We have previously modeled bulk gene expression data with denoising autoencoders and non-linear activation functions [4–6].
+These models have engineered features that encode information about gene ontology terms and KEGG pathways.
+We have shown that these models capture more information than linear models like PCA, ICA, and NMF, and that the learned features are often overlapping and redundant [7].
+We have also recently trained a VAE on cancer gene expression datasets and observed that the learned latent features capture interesting biological distributions of features across different tumors [8].
+Therefore, we are confident that modeling gene expression with reconstruction cost captures meaningful biology.
+Furthermore, we trained VAEs and GANs on electronic health record (EHR) data including a model to generate privacy aware fake datasets to supplement learning or data sharing [9,10].
+We will develop similar approaches to extract knowledge from Human Cell Atlas (HCA) single cell data.
+
+## Enabling latent space arithmetic in single cell gene expression with variational autoencoders:
```

Comment:
I will update the proposed experiment, eval strategy, and limitations in the next commit